azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -1872,7 +1872,11 @@ private void processQueuedFlows(long activeExecutorsRefreshWindow,
           // process flow with current snapshot of activeExecutors
           selectExecutorAndDispatchFlow(reference, exflow, new HashSet<Executor>(activeExecutors));
         }
-        currentContinuousFlowProcessed++;
+
+        // do not count failed flow processsing (flows still in queue)
+        if(queuedFlows.getFlow(exflow.getExecutionId()) == null) {
+          currentContinuousFlowProcessed++;
```

Comment:
I am curious, shouldn't 
"currentContinuousFlowProcessed++;"
be added right after line 1873?

otherwise we will count 1 extra when a exflow wakes up from the sleep section,.even though it hasn't been assigned